src/cpp/server/db/20200226141952248_AddRevokedCookie.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE RevokedCookie(
+   Expiration text NOT NULL,
+   CookieData text NOT NULL,
+   PRIMARY KEY (Expiration, CookieData)
```

Comment:
I want the expiration on its own column so we can use the database to filter for the bulk delete query. Having the composite key here is going to be negligible performance loss (and this is not super performance critical code at any rate), so I'd rather keep both columns.

Reply:
Oh, I didn't mean to remove the expiration column, just to remove it from the primary key if it's not required for unique entry identification. You're right about the performance though - that wouldn't be the reason to change it.

Reply:
That's a good point - since the Expiration is derivative data (it's redundantly stored on `CookieData`), then there's no reason for it to be part of the primary key. It just made sense in my initial thoughts. I'll change this.

Reply:
Actually, I do want the `Expiration` partially indexed for faster sorting (since we sort by expiration). I'll need to research more on how compound primary keys affect this.

Reply:
Are you sure it's a partial index that you want, or a full index? At any rate you should be able to create an index on a non-primary-key column using `CREATE INDEX RCExpiryIndex ON RevokedCookies (Expiration);` or something like it. It makes insertion into the table slower though, so if you're going to do that you may want to revise the code that writes all the revoked cookies to the table (assuming that happens with any frequency).

Reply:
The overhead that this index is going to add, especially for such a small table that doesn't see a very high volume, is minimal and not worth reworking any of the insert code IMO.